notebooks/simone_pipeline.md

```diff
@@ -0,0 +1,134 @@
+# Summary of Simone's pipeline
+
+## Overview of methods needed
+
+1. 3d gaussian filter (`skimage.filters.gaussian` or `scipy.ndimage.gaussian`) -> `starfish.filters.gaussian` (already present)
+2. `ndimage.gaussian_laplace` -> `starfish.filters.gaussian_laplace`
```

Comment:
there's also: http://scikit-image.org/docs/dev/api/skimage.filters.html#skimage.filters.laplace. I *think these both implement a laplacian of gaussian (LoG) filter? can you try both and see if they're actually the same?

Reply:
I think the function you linked is more similar to https://github.com/scipy/scipy/blob/v0.16.1/scipy/ndimage/filters.py#L397 than to https://github.com/scipy/scipy/blob/v0.16.1/scipy/ndimage/filters.py#L413 (what the osmFISH pipeline is using). 

The functions are definitely not the same, but it's possible they're similar enough to use, or that a preprocessing step could make them equivalent. the skimage function is _much_ faster. (3s vs 50s).